Dockerfile

```diff
@@ -75,13 +75,13 @@ WORKDIR /
 RUN echo "===> install Kaldi (pinned at version 5.3)"  && \
     git clone -b 5.3 https://github.com/kaldi-asr/kaldi && \
     cd /kaldi/tools && \
-    make && \
+    make -j`nproc` && \
     ./install_portaudio.sh && \
     cd /kaldi/src && ./configure --mathlib=ATLAS --shared  && \
     sed -i '/-g # -O0 -DKALDI_PARANOID/c\-O3 -DNDEBUG' kaldi.mk && \
-    make depend && make && \
-    cd /kaldi/src/online2 && make depend && make && \
-    cd /kaldi/src/online2bin && make depend && make
+    make depend -j`nproc` && make -j`nproc` && \
+    cd /kaldi/src/online2 && make depend -j`nproc` && make -j`nproc` && \
+    cd /kaldi/src/online2bin && make depend -j`nproc` && make -j`nproc`
```

Comment:
nproc fails in docker hub builds (and low resource local builds) due to running out of memory. As such we aren't able to support using the `-j` flag on kaldi.

Reply:
Done, I made a double top line with a comment to easily allow both ways